docs/standards/universal-profile/lsp0-erc725account.md

```diff
@@ -13,153 +13,235 @@ sidebar_position: 2
+
+When the owner of the account is an EOA, the **ECDSA algorithm** is used to recover the address of the signer from the provided signature and message, and the function `isValidSignature(..)` will return **valid** if the recovered signer address matches the address of the owner.
+
+When the owner is a smart contract, the `isValidSignature(..)` function will be called on the owner and return whether the signature and the message are **valid** according to the logic in `isValidSignature(..)` on the owner contract.
+
 ### LSP1 - UniversalReceiver
 
-:::info
+:::tip
 
 See the **[LSP1-UniversalReceiver](../generic-standards/lsp1-universal-receiver.md)** standard for more information.
 
+Check the [**universalReceiver functions**](../smart-contracts/lsp0-erc725-account.md#universalreceiver) provided by **LSP1** that allows notifying about incoming/ outgoing transactions.
+
+Check the **javascript** guides to know [**How to set a UniversalReceiverDelegate built by LUKSO**](../../guides/universal-receiver-delegate/set-default-implementation.md) or [**How to set your own UniversalReceiverDelegate**](../../guides/universal-receiver-delegate/accept-reject-assets.md).
+
 :::
 
 This standard enables the account to be notified of incoming transactions such as token transfer, vault transfer, information transfer, etc. Notifications are handy for situations where users want to customize how their account contract reacts to certain tokens by either rejecting them or operating a specific call on each token received.
 
 The **[LSP0-ERC725Account](#)** implements the `universalReceiver(..)` function that:
 
-- Emits an event with the typeId and data passed to it, as well as additional parameters such as the amount sent to the function, the caller of the function, and the return value of the delegate contracts.
+Emits an event with the `typeId` and `data` passed to it, as well as additional parameters such as the amount sent to the function, the caller of the function, and the return value of the delegate contracts.
 
-- Forwards the call to the **UniversalReceiverDelegate** contract address stored under the data key attached below, if it supports [LSP1UniversalReceiverDelegate InterfaceId](../smart-contracts/interface-ids.md).
+The `typeId` is a **bytes32** that represents the type of action being notified about. For instance, if you want to notify an account about a specific type of token, you could hash the word **"TOKENXX"** which would result in a bytes32, and then use it as the typeId.
 
-```json
-{
-  "name": "LSP1UniversalReceiverDelegate",
-  "key": "0x0cfc51aec37c55a4d0b1a65c6255c4bf2fbdf6277f3cc0730c45b828b6db8b47",
-  "keyType": "Singleton",
-  "valueType": "address",
-  "valueContent": "Address"
-}
-```
+The `data` field would contain relevant information related to the typeId being notified about, in case of token it could be any encoded information such as its balance, time, royalties, etc.
```

Comment:
```suggestion
The `data` field can contain relevant information related to the `typeId` used when the `universalReceiver(...)` function was called. In the case of token transfers, it could be any encoded information such as the recipient balance, time, royalties, etc.
```